### PR TITLE
Product switch price formatting 

### PR DESCRIPTION
**Pull request by charleycampbell**
## What does this change?
On Switch Options, Switch Complete and Switch Review, where it shows a whole monetary amounts in headers and text any trailing 0s are removed, and decimals are rounded to two places. For example £5.90 and £5.

## How to test
Any monetary value should appear in the Switch Option, Switch Complete  and Switch Review Storybook test. Since this remains behind a feature switch it can't be tested in production but on CODE (if the feature flag has been lifted) follow the product switch journey and you will see the value is fixed to two decimal places on the switch review page and the switch complete page.

## Images
**Switch Option** 
image

image

**Switch Review**
image

**Switch Complete**
image

## Accessibility

-   [ ] [Tested with screen reader](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#screen-reader)
-   [ ] [Navigable with keyboard](https://github.com/guardian/accessibility/blob/main/people-and-technology/02-physical.md#keyboard)
-   [ ] [Colour contrast passed](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#contrast)
-   [ ] [The change doesn't use only colour to convey meaning](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#use-of-colour)

### CODE REVIEW

**Comment by charleycampbell** [comment]
[Product Switching:  Price Formatting](https://trello.com/c/qHFsWgZf/3848-product-switching-price-formatting)

**Comment by charleycampbell** [comment]
[Post design review updates: tranche 2](https://trello.com/c/hSjROh5J/3832-post-design-review-updates-tranche-2)

File: client/components/mma/switch/SwitchOptions.tsx [hunk 1]
```diff
@@ -163,7 +172,7 @@ export const SwitchOptions = () => {
 							You're currently supporting the Guardian with a{' '}
 							{monthlyOrAnnual.toLowerCase()} contribution of{' '}
 							{mainPlan.currency}
-							{currentAmount}.
+							{currentAmount.toFixed(2)}.
```
Comment (andrewHEguardian) [inline client/components/mma/switch/SwitchOptions.tsx:175]: this could also be an integer - seems worth checking all prices before doing toFixed(2)

Comment (charleycampbell) [reply client/components/mma/switch/SwitchOptions.tsx:175]: Yeah hence why I've popped this back into draft

**Comment by charleycampbell** [comment]
If we go ahead with this change there would be refactor work to pull the function out and import it in

**Pull request merged**